.travis.yml

```diff
@@ -55,6 +55,11 @@ before_script:
 script:
 - docker-compose build --pull app-dev-base;
 - docker-compose -f docker-compose.yml -f docker-compose.prod.yml build --pull app-prod;
+- for i in $(seq 1 5); do
```

Comment:
see my comment on gitlab-ci to apply for this (use app-dev instead for faster feedback)